tldw_Server_API/app/services/registration_service.py

```
@@ -268,6 +268,7 @@ async def register_user(
         is_active_override: Optional[bool] = None,
         is_verified_override: Optional[bool] = None,
         storage_quota_override: Optional[int] = None,
+        system_provisioning: bool = False,
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)